tests/test_typings.py

```diff
@@ -11,13 +11,16 @@
 class MyPyTest(TestCase):
     """Class for testing modules with mypy."""
 
+
+    @unittest.skip("Causing out of memory error on travis")
```

Comment:
Does this happen anywhere but on travis?

Reply:
Good question, supposedly travis gives us 7.5GB ... would be interesting to know if mypy was using that much but still eventually getting the job done on a system with more memory, vs going into a non-terminating leak of some sort.

Reply:
I don't have a problem on my laptop. I fix any typing suggestions and give it another try on travis.

Reply:
I fixed the typing suggestions and tried it on travis and it failed on travis. I'm going to set them to skipped again.